README.md

```diff
@@ -64,40 +65,102 @@ Flags:
 ```
 
 Every argument can be defined using environment variables using `ROSETTA_` prefix; and replacing `.` for `_`; for example:
-```
+
+```sh
 ROSETTA_DATADIR="/my/dir"
 ROSETTA_GETH_GENESIS="/path/to/genesis.json"
-``` 
+```
+
+## Running the Rosetta RPC Server
 
-## Running Rosetta Docker Image
+Running the Rosetta RPC Server from scratch will take some time to sync, since it runs a full archive node in the background. While it may be possible to run the Construction API in the future with a non-archive node, this is still required by the Rosetta spec for the Data API implementation in order to perform balance reconciliation.
 
-Rosetta is released as a docker image: `us.gcr.io/celo-testnet/rosetta`. All version can be found on the [registry page](https://us.gcr.io/celo-testnet/rosetta)
+### Version 1: Running from `rosetta` source code
 
-Within the docker image, we pack `rosetta` binary and also `geth` binary from celo-blockchain. Rosetta will run both.
+You will need the following three repositories cloned locally: `rosetta` (this repo), [`celo-monorepo`](https://github.com/celo-org/celo-monorepo), [`celo-blockchain`](https://github.com/celo-org/celo-blockchain). You also need: `go >= 1.14`, `rust >= 1.41.0` (blockchain dependency)
 
-To run Rosetta using the docker container, the following options must be configured:
-  * `genesis.json` for the target network (can be found by `curl 'https://storage.googleapis.com/genesis_blocks/baklava' > genesis.json`)
-  * `staticNodes` or `bootnodes`.
-    * With `staticNodes` Rosetta will directly peer to the list of staticNode provided. This node can be any you have access to. For a public list check `https://storage.cloud.google.com/static_nodes/baklava`
+#### Running on Alfajores (Testnet)
 
-Additionaly, it needs a data directory for the geth datadir & rosetta.db
+Prerequisites:
+
+* Checkout `celo-monorepo` branch `alfajores` and run `yarn && yarn build`
+* Checkout `celo-blockchain` tag `v1.1.2` (NOTE: check that this matches the version specified in the `rosetta` `go.mod` file) and `make all`
+* Export paths to `celo-monorepo` and `celo-blockchain` as `CELO_MONOREPO_PATH` and `CELO_BLOCKCHAIN_PATH` respectively (can be paths relative to `rosetta` repo)
+* Checkout `rosetta` tag `v0.7.6` (or latest released tag) and `make gen-contracts && make all`
```

Comment:
Checkout `rosetta` tag `v0.7.6` (or latest released tag)

```sh
$ git fetch --all --tags
$ git checkout tags/v0.7.6 -b v0.7.6-branch
```

Reply:
✅ added a line for this